src/components/ViewMenu.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Table } from "semantic-ui-react";
+import { getMenu } from "../modules/menu";
+import { Component } from "react";
+
+class ViewMenu extends Component {
+  state = {
+    menu: [],
+  };
+  componentDidMount() {
+    this.getViewMenu();
+  }
+  async getViewMenu() {
+    let result = await getMenu();
+    this.setState({ menu: result.data.products });
+  }
+  render() {
+    let menu;
+    if (this.state.menu != null) {
+      menu = this.state.menu.map((product) => {
+        return (
+          <Table.Row key={product.id}>
+            <Table.Cell>{product.title}</Table.Cell>
+            <Table.Cell>{product.description}</Table.Cell>
+            <Table.Cell>{product.price} $</Table.Cell>
+            <Table.Cell>
+              <button id="minus">-</button>
```

Comment:
These buttons are out of scope for this feature. Why have buttons that do nothing?